### PR TITLE
Toradex support on Zeus.

### PR DESCRIPTION
**Pull request by drewmoseley**
So far just eMMC based builds.

### CODE REVIEW

**Comment by drewmoseley** [comment]
Added raw NAND support.

File: meta-mender-toradex-nxp/recipes-bsp/mender-tezi-metadata/mender-tezi-metadata_1.0.bb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+DESCRIPTION = "Toradex Easy Installer Metadata"
+
+inherit deploy
+
+SRC_URI = " \
+    file://LICENSE \
```
Comment (mirzak) [inline meta-mender-toradex-nxp/recipes-bsp/mender-tezi-metadata/mender-tezi-metadata_1.0.bb:6]: The LICENSE text seems to be generic so should be enough to use:

```
LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

Then you can remove this file.

File: meta-mender-toradex-nxp/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This is the Toradex machine layer for meta-mender-community.
```
Comment (mirzak) [inline meta-mender-toradex-nxp/README.md:1]: Please update this to the more common format in meta-mender-community.

Example: https://github.com/mendersoftware/meta-mender-community/tree/zeus/meta-mender-odroid

**Review by mirzak — COMMENTED** [review]
Overall looks good. Great work!

Have two suggestions.

**Comment by drewmoseley** [comment]
@mirzak all comments address and force-pushed.
Also I created the Mender Hub entry for Apalis: https://hub.mender.io/t/toradex-nxp-i-mx-6-computer-on-module-apalis-imx6/2331

**Review by mirzak — APPROVED** [review]

**Pull request merged**